### PR TITLE
Update Dacfx 18.7.1

### PR DESCRIPTION
**Pull request by dzsquared**
# Description
New tool, Bug fixing, or Improvement?  Improvement, Updating DacFx to latest version 18.7.1

#### Related issue:
https://github.com/actions/virtual-environments/issues/3607

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/Windows2016-Readme.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ Note: MSYS2 is pre-installed on image but not added to PATH.
 
 ### Database tools
 - Azure CosmosDb Emulator 2.11.13.0
-- DacFx 15.0.4897.1
+- DacFx 15.0.5164.1
```
Comment (al-cheb) [inline images/win/Windows2016-Readme.md:208]: Please don't update auto generating documentation file.

Comment (dzsquared) [reply images/win/Windows2016-Readme.md:208]: ah, apologies

Comment (al-cheb) [reply images/win/Windows2016-Readme.md:208]: @dzsquared, Pls, remove any changes related to *.md files from the PR.

Comment (dzsquared) [reply images/win/Windows2016-Readme.md]: @al-cheb finally had a chance to clean up that small mess

**Review by al-cheb — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**